Tests/L0/SonarQubePostTest/data/ReportTest/metricsResponse.json

```diff
@@ -0,0 +1,34 @@
+{
+    "metrics": [
+        {
+            "id": "11",
+            "key": "accessors",
+            "type": "INT",
+            "name": "Accessors"
+        },
+        {
+            "id": "93",
+            "key": "blocker_violations",
+            "type": "INT",
+            "name": "Blocker issues"
+        },
+        {
+            "id": "88",
+            "key": "duplicated_blocks",
+            "type": "INT",
+            "name": "Duplicated blocks"
+        },
+        {
+            "id": "108",
+            "key": "sqale_index",
+            "type": "WORK_DUR",
+            "name": "Technical Debt"
+        },
+        {
+            "id": "109",
+            "key": "new_technical_debt",
+            "type": "WORK_DUR",
```

Comment:
Note that I could add the unit of measurement to the metrics. However I fear we will be too closely coupled with SonarQube in this case. I will discuss with Jean-Marc further.